extension/src/extension.ts

```diff
@@ -1,32 +1,35 @@
 import {
-    ExaAllScripts,
+    ExaMetadata,
     ExasolExtension,
     Installation,
     Instance, ParameterValues,
     registerExtension,
     SqlClient
 } from "@exasol/extension-manager-interface";
+import { Context } from "./common";
 import { CONFIG } from "./extension-config";
+import { findInstallations } from "./implementation";
 
 export function createExtension(): ExasolExtension {
     const version = CONFIG.version;
-    const filename = CONFIG.fileName;
+    const fileName = CONFIG.fileName;
     const fileSize = CONFIG.fileSizeBytes;
     const repoBaseUrl = "https://github.com/exasol/s3-document-files-virtual-schema"
-    const downloadUrl = `${repoBaseUrl}/releases/download/${version}/${filename}`;
+    const downloadUrl = `${repoBaseUrl}/releases/download/${version}/${fileName}`;
```

Comment:
Am I correct, that repoBaseUrl is used only for licenseUrl? 
If so then IMHO we could as well just create a variable ``licenseUrl`` and just use it.